megamek/src/megamek/common/weapons/CapitalMissileHandler.java

```diff
@@ -305,39 +338,54 @@ public boolean handle(IGame.Phase phase, Vector<Report> vPhaseReport) {
      */
     @Override
     protected int calcAttackValue() {
+        AmmoType atype = (AmmoType) ammo.getType();
         int av = 0;
         double counterAV = calcCounterAV();
         int armor = wtype.getMissileArmor();
-        // if we have a ground firing unit, then AV should not be determined by
-        // aero range brackets
-        if (!ae.isAirborne() || game.getOptions().booleanOption(OptionsConstants.ADVCOMBAT_UAC_TWOROLLS)) {
-            if (usesClusterTable()) {
-                // for cluster weapons just use the short range AV
-                av = wtype.getRoundShortAV();
+        //AR10 munitions
+        if (atype != null) {
+            if (atype.getAmmoType() == AmmoType.T_AR10) {
+                if (atype.hasFlag(AmmoType.F_AR10_KILLER_WHALE)) {
+                    av = 4;
```

Comment:
I'm curious, is this something that can be put into the ammo type definition instead?

Reply:
I think this is as it is due to handling nuclear warheads.  I'm inclined to leave it alone at the moment, but since there's more to do with different missile warheads and WMDs I can revisit the topic when I get to those.